tests/test_order_page.py

```diff
@@ -0,0 +1,20 @@
+import allure
+
+
+class TestOrderFeed:
```

Comment:
Нужно исправить: тут должна быть проверка создания заказа залогиненым пользователем. На ui а не по апи. Это ui проект

Reply:
HI Alex,
    Проверка основного функционала(залогиненный пользователь может оформить заказ.) - 
    is being tested in test_main_page class TestMainPage within the following function:
    **@allure.title("Verify authorised user creates order")
    def test_user_create_order(self, get_driver):**
    
 I have refactored this test to ensure it runs entirely through the UI, instead of using API calls, as per your feedback. 
 
 Thank you.